doctor.py

```diff
@@ -67,25 +67,29 @@ def main(context: Dict[str, str], argv: Sequence[str] | None = None) -> int:
         if args.check_only:
             return 1
 
-    print("\nAttempting to run fixes for the checks that failed...")
-    futures = {}
-    results = {}
+    print("\nThe following checks need to be fixed:")
+    skip = []
     for check in retry:
-        futures[check] = executor.submit(check.fix)
-    for check, future in futures.items():
-        results[check] = future.result()
-
-    for check, result in results.items():
-        ok, msg = result
-        if ok:
-            print(f"✅ fix: {check.name}")
-            continue
-        print(f"❌ fix: {check.name}{msg}")
+        print(f"❌ {check.name}")
+        if input(f"Do you want to attempt to fix {check.name}? (y/n): ").lower() == "y":
```

Comment:
Optional: Can we add a default value of y and change the input to `(Y/n)` ?

Reply:
sure!